libs/windy-sounding/types/client/ExternalSveltePlugin.d.ts

```
@@ -45,9 +46,14 @@ export declare class ExternalSveltePlugin extends SveltePlugin<'windy-external-p
       listenToSingleclick,
       addToContextmenu,
       url,
+      version,
     }: InstalledExternalPluginConfig,
   );
   open({ params, disableOpeningAnimation, qs }: PluginOpeningOptions<'windy-external-plugin'>): Promise<void | boolean>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)